meta-mender-raspberrypi/recipes-bsp/u-boot/patches/0001-configs-rpi-enable-mender-requirements.patch

```diff
@@ -42,11 +42,11 @@ index 75c6c9c447..493a99a26d 100644
  CONFIG_OF_LIBFDT_OVERLAY=y
  CONFIG_EFI_LOADER=y
 +CONFIG_ENV_IS_IN_MMC=y
-+CONFIG_ENV_OFFSET=0x800000
-+CONFIG_ENV_OFFSET_REDUND=0x1000000
++CONFIG_ENV_OFFSET=0x400000
++CONFIG_ENV_OFFSET_REDUND=0x800000
```

Comment:
This doesn't look correct to me, this was [just fixed the opposite way](https://github.com/mendersoftware/meta-mender/commit/3d366e21cec4b6e15faa4febcedc430cdd9eb546) because zeus would not work otherwise. What's the reason for this patch?

Reply:
At this point I am only doing build time tests so it's very possible this is wrong.  I'll resync my sources and see if this is no longer needed.